rpcs3/util/asm.hpp

```diff
@@ -7,7 +7,7 @@
 extern bool g_use_rtm;
 extern u64 g_rtm_tx_limit1;
 
-#ifdef _M_X64
+#ifdef _MSC_VER
```

Comment:
I think this breaks arm64 builds.

Reply:
Do you use MSVC for arm64 builds?

Reply:
```suggestion
#if defined(_M_X64) && defined(_MSC_VER)
```

Reply:
immintrin.h stuff is x86 only, arm64 should not have it.

Reply:
I'll revert this change, it was only necessary for Clang. BTW Microsoft [says](https://learn.microsoft.com/en-us/cpp/intrinsics/x86-intrinsics-list?view=msvc-170) that it's a 3DNOW intrinsic.